.gitmodules

```diff
@@ -4,7 +4,8 @@
   ignore = dirty
 [submodule "sorc/wxflow"]
   path = sorc/wxflow
-  url = https://github.com/NOAA-EMC/wxflow
+  url = https://github.com/TerrenceMcGuinness-NOAA/wxflow
+  branch = config_env_xor
```

Comment:
We need to get this into the authoritative wxflow before we can merge this.

Reply:
Correct.  I have in the notes this is pending on passing the CI tests and then I'll put in a PR in wxflow.

Reply:
Removed the pending wxflow PR with config_env_xor branch from this PR.